pom.xml

```diff
@@ -62,6 +62,7 @@
     <armeria.version>0.84.0</armeria.version>
     <!-- This is from armeria, but be careful to avoid >= v20 apis -->
     <guava.version>27.0.1-jre</guava.version>
+    <netflix.concurrency.limits.version>0.2.0</netflix.concurrency.limits.version>
```

Comment:
please move to call-site. we drifted this so it is only needed at the zipkin-server pom. probably doesn't need a variable.